.docker/images/govcms8/scripts/govcms-deploy

```diff
@@ -20,6 +20,15 @@ mkdir -p /app/web/sites/default/files/private/tmp/
 config_count=`ls -1 /app/config/default/*.yml 2>/dev/null | wc -l`
 dev_config_count=`ls -1 /app/config/dev/*.yml 2>/dev/null | wc -l`
 
+# Database options to configure the remote to use the read replica when
+# performing read operations.
+sync_opts=""
+read_replica_enabled () {
+  if tables=$(drush @govcms-prod sqlq 'show tables; --database=read') && [ -n "$tables" ]; then
```

Comment:
```suggestion
  if tables=$(drush @govcms-prod sqlq 'show tables;' --database=read) && [ -n "$tables" ]; then
```

Reply:
Bail early if env var is not present; prevent a failed bootstrap.

Redirect stderr away, we do not need to see failures here (but should prob echo out so we know which db is being used for sync)